src/assets/wise5/components/component-student.component.ts

```diff
@@ -115,6 +116,22 @@ export abstract class ComponentStudent {
     );
   }
 
+  subscribeToNotifyConnectedComponents(): void {
+    this.subscriptions.add(
+      this.ComponentService.notifyConnectedComponentSource$.subscribe(
+        ({ nodeId, componentId, componentState }) => {
+          if (this.nodeId === nodeId && this.componentId === componentId) {
```

Comment:
```suggestion
          if (nodeId === this.nodeId && componentId === this.componentId) {
```

Reply:
I usually write it with this.nodeId on the left. Would you write
```this.nodeId === 'node1'```
or
```'node1' === this.nodeId```
?
For me, the first method of doing it reads more like regular English. Would you say "is my age equal to 10?" or "is 10 equal to my age?"?

Reply:
Hmm. Doesn't the variable usually go on the left side of the ```===``` and the constant go on the right? In this case, the variable is ```nodeId```, since it can be different each time, and the constant is ```this.nodeId```, since it remains the same for this component?